reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -288,12 +308,14 @@ Administrator.
 Invoke-Command -ComputerName Server01 -Credential Domain01\Admin01
 ```
 
-For more information about the Credential parameter, see New-PSSession,
-Enter-PSSession or Invoke-Command.
+For more information about the Credential parameter, see `New-PSSession`,
```

Comment:
Shouldn't a "For more information" be links to docs instead of code fences on cmdlets?

Reply:
Fixed